### PR TITLE
fix: Fix range bounds in QueryMsg::Emission

### PR DESCRIPTION
**Pull request by apollo-sturdy**

### CODE REVIEW

**Review by piobab — APPROVED** [review]

**Comment by pacmanifold** [comment]
@apollo-sturdy This was the fix i though should work. How ever i tried querying the current contract with `current_time - epoch_duration` and it still returned 0. Only when providing exactly the right timestamp did it return non-zero

Have you tried to see if this works?

File: contracts/incentives/tests/test_quering.rs [hunk 1]
```diff
@@ -143,6 +143,28 @@ fn query_emission() {
...
+    // Query at timestamp some time after emission end
+    let res: Uint128 = helpers::th_query(
+        deps.as_ref(),
+        QueryMsg::Emission {
+            collateral_denom: "uatom".to_string(),
+            incentive_denom: "umars".to_string(),
+            timestamp: 604800 * 3,
+        },
+    );
+    assert_eq!(res, Uint128::zero());
```
Comment (pacmanifold) [inline contracts/incentives/tests/test_quering.rs:167]: lets add one test case querying before emission start

**Review by pacmanifold — APPROVED** [review]
one comment. otherwise lgtm

**Review by pacmanifold — APPROVED** [review]
lgtm

**Pull request merged**